packages/@o3r/rules-engine/src/components/rules-engine/rule-condition/rule-condition-pres.component.ts

```diff
@@ -35,18 +39,19 @@ export class RuleConditionPresComponent {
    */
   @Input()
   public set condition(condition: TopLevelCondition | undefined) {
+    // eslint-disable-next-line no-underscore-dangle -- false positive
```

Comment:
It is more that it is to identify a private object

Reply:
after the rebase this issue should not be raised anymore